src/javascript/app/pages/trade/markets.jsx

```diff
@@ -68,11 +68,19 @@ const List = ({
 class Markets extends React.Component {
     constructor (props) {
         super(props);
-        let market_symbol = Defaults.get('market');
+        let market_symbol,underlying_symbol;
+        const login_id = localStorage.getItem('active_loginid');
+        if (localStorage.getItem(`${login_id}_selected_market`) && login_id) {
+            market_symbol = localStorage.getItem(`${login_id}_selected_market`);
```

Comment:
you are retrieving this value twice, better store it in a variable instead

Reply:
done